### PR TITLE
[CELEBORN-500][WEBSITE] Add migration guide

### PR DESCRIPTION
**Pull request by waitinfuture**
image

### CODE REVIEW

**Comment by waitinfuture** [comment]
ping @pan3793 please take a look

File: docs/docs/latest/migration.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Migration Guide
+
+## Upgrading from 0.2.1 to 0.3.0
+From 0.3.0 on the default value for celeborn.push.replicate.enabled is changed from true to false, users
```
Comment (pan3793) [inline docs/docs/latest/migration.md]:
```suggestion

From 0.3.0 on the default value for `celeborn.push.replicate.enabled` is changed from `true` to `false`, users
```

File: docs/docs/latest/migration.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+# Migration Guide
+
+## Upgrading from 0.2.1 to 0.3.0
+
+From 0.3.0 on the default value for `celeborn.push.replicate.enabled` is changed from `true` to `false`, users
+who want replication on should explicitly enable replication. For example, to enable replication for Spark
+users should add the spark config when submitting job:
+spark.celeborn.push.replicate.enabled=true
```
Comment (pan3793) [inline docs/docs/latest/migration.md]:
```suggestion
users should add the spark config when submitting job: `spark.celeborn.push.replicate.enabled=true`
```

File: mkdocs.yml [hunk 3]
```diff
@@ -90,6 +90,7 @@ nav:
           - Monitoring: docs/latest/monitoring.md
           - Upgrade: docs/latest/upgrade.md
           - Ratis Shell: docs/latest/celeborn_ratis_shell.md
+          - Migration Guid: docs/latest/migration.md
```
Comment (pan3793) [inline mkdocs.yml]:
```suggestion
          - Migration Guide: docs/latest/migration.md
```

**Review by pan3793 — APPROVED** [review]

**Pull request merged**